makefiles/pacman.mk

```
@@ -3,27 +3,44 @@ $(error Use the main Makefile)
 endif
 
 SUBPROJECTS    += pacman
-PACMAN_VERSION := 5.2.2
-DEB_PACMAN_V   ?= $(PACMAN_VERSION)-2
+PACMAN_VERSION := 6.0.0
+DEB_PACMAN_V   ?= $(PACMAN_VERSION)
 
 pacman-setup: setup
 	wget -q -nc -P $(BUILD_SOURCE) https://git.archlinux.org/pacman.git/snapshot/pacman-$(PACMAN_VERSION).tar.gz
 	$(call EXTRACT_TAR,pacman-$(PACMAN_VERSION).tar.gz,pacman-$(PACMAN_VERSION),pacman)
 	$(call DO_PATCH,pacman,pacman,-p1)
+	mkdir -p $(BUILD_WORK)/pacman/build
+	echo -e "[host_machine]\n \
+	system = 'darwin'\n \
+	cpu_family = '$(shell echo $(GNU_HOST_TRIPLE) | cut -d- -f1)'\n \
+	cpu = '$(MEMO_ARCH)'\n \
+	endian = 'little'\n \
+	[properties]\n \
+	root = '$(BUILD_BASE)'\n \
+	[paths]\n \
+	prefix ='$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)'\n \
+	sysconfdir='$(MEMO_PREFIX)/etc'\n \
+	localstatedir='$(MEMO_PREFIX)/var'\n \
+	[binaries]\n \
+	c = '$(CC)'\n \
+	cpp = '$(CXX)'\n" > $(BUILD_WORK)/pacman/build/cross.txt
 
 ifneq ($(wildcard $(BUILD_WORK)/pacman/.build_complete),)
 pacman:
 	@echo "Using previously built pacman."
 else
-pacman: pacman-setup libarchive openssl curl gettext
-	cd $(BUILD_WORK)/pacman && ./autogen.sh
-	cd $(BUILD_WORK)/pacman && ./configure \
-		$(DEFAULT_CONFIGURE_FLAGS) \
-		--disable-dependency-tracking \
-		--disable-doc
-	+$(MAKE) -C $(BUILD_WORK)/pacman
-	+$(MAKE) -C $(BUILD_WORK)/pacman install \
-		DESTDIR=$(BUILD_STAGE)/pacman
+pacman: pacman-setup libarchive openssl curl gettext gpgme bash-completion
+	cd $(BUILD_WORK)/pacman/build && PKG_CONFIG="pkg-config" meson \
+		--cross-file cross.txt \
+		--buildtype=release \
+		-D i18n=false \
+		-D doc=disabled \
+		-D crypto=openssl \
+		-D b_bitcode=true \
+		..
+	+ninja -C $(BUILD_WORK)/pacman/build
+	+DESTDIR="$(BUILD_STAGE)/pacman" ninja -C $(BUILD_WORK)/pacman/build install
```

Change line 43 to the following
```suggestion
	+ninja -C $(BUILD_WORK)/pacman/build install \
		DESTDIR="$(BUILD_STAGE)/pacman"
```